test/tools/unified-spec-runner/entities.ts

```diff
@@ -242,7 +242,7 @@ export class FailPointMap extends Map<string, Document> {
         try {
           await client.connect();
         } catch (error) {
-          console.error(`failed to connect disableFailPoints ${hostAddress}`);
+          console.error(`failed to connect disableFailPoints ${hostAddress} - ${error}`);
```

Comment:
Just curious - does that actually work well? in my experience the error objects don't do well inside template strings, and also we throw the error right away, so hypothetically, shouldn't that make the error show up in the logs immediately after that console statement?

Reply:
For some reason nothing was showing up, but this is easily re-added if debugging comes up again